internal/pkg/service/stream/components.go

```diff
@@ -59,7 +59,7 @@ func StartComponents(ctx context.Context, d dependencies.ServiceScope, cfg confi
 	}
 
 	if componentsMap[ComponentHTTPSource] {
-		d, err := dependencies.NewSourceScope(d, cfg)
+		d, err := dependencies.NewSourceScope(d, "http-source", cfg)
 		if err != nil {
 			return err
```

Comment:
We need source type here
- to create distribution group
- to know about other http source nodes
- to assign volumes/slices on disk writer nodes to http source nodes, for writing

Reply:
Yes